docs/4.1/admin-guide.md

```diff
@@ -2125,25 +2119,24 @@ To configure Teleport to use DynamoDB:
 teleport:
   storage:
     type: dynamodb
-    region: eu-west-1
+    # Region location of dynamodb instance, https://docs.aws.amazon.com/en_pv/general/latest/gr/rande.html#ddb_region 
+    region: us-east-1
 
     # Name of the DynamoDB table. If it does not exist, Teleport will create it.
-    table_name: teleport_table
-
-    # Authentication settings are optional (see below)
-    access_key: BKZA3H2LOKJ1QJ3YF21A
-    secret_key: Oc20333k293SKwzraT3ah3Rv1G3/97POQb3eGziSZ
+    # table_name is 
+    table_name: Example_TELEPORT_DYNAMO_TABLE_NAME
+    audit_table_name: Example_TELEPORT_DYNAMO_EVENTS_TABLE_NAME
 
     # This setting configures Teleport to send the audit events to three places: 
     # To keep a copy on a local filesystem, in DynamoDB and to Stdout. 
     audit_events_uri:  ['file:///var/lib/teleport/audit/events', 'dynamodb://table_name', 'stdout://']
```

Comment:
(this is the 'new' way of storing events to DynamoDB)

Reply:
Sorry @webvictim not sure if this a note / comment. If I should update let me know.